aioesphomeapi/api.proto

```diff
@@ -2310,11 +2310,13 @@ message ZWaveProxyFrame {
 enum ZWaveProxyRequestType {
   ZWAVE_PROXY_REQUEST_TYPE_SUBSCRIBE = 0;
   ZWAVE_PROXY_REQUEST_TYPE_UNSUBSCRIBE = 1;
+  ZWAVE_PROXY_REQUEST_TYPE_HOME_ID_CHANGE = 2;
 }
 message ZWaveProxyRequest {
   option (id) = 129;
-  option (source) = SOURCE_CLIENT;
+  option (source) = SOURCE_BOTH;
   option (ifdef) = "USE_ZWAVE_PROXY";
 
   ZWaveProxyRequestType type = 1;
+  bytes data = 2;
```

Comment:
probably could use the zero copy here

Reply:
🤦🏻‍♂️  Why yes, yes I should...one min. haha

Reply:
True, it's only very rarely used...haha

Reply:
[Done](https://github.com/esphome/aioesphomeapi/pull/1371/commits/347505dbd6c20dd47f553ce059bbc06c9a0226d3)